.gitignore

```diff
@@ -80,6 +80,7 @@ venv.bak/
 *.idea
 *.vscode
 *.iml
+*.devcontainer
```

Comment:
```suggestion
```

Reply:
you don't want to preemptively support ignoring the containerized dev environment for VScode?

Reply:
I thought you pushed this change by mistake. Can we file a separate PR?

Reply:
sure thing!